app/javascript/dashboard/routes/dashboard/settings/integrations/DashboardApps/DashboardAppsRow.vue

```diff
@@ -43,3 +49,15 @@ export default {
   },
 };
 </script>
+
+<style lang="scss" scoped>
+.dashboard_app-url {
+  max-width: calc(var(--space-giga) + var(--space-mega));
```

Comment:
@fayazara This doesn't seem to be the right fix. What would happen on smaller/larger screens?

Reply:
@pranavrajs Another option would be to add `table-layout: auto` and `width:100%`, so words can overflow. 

Which would look something like this, 
<img width="1190" alt="Screenshot2022-08-09 at 15 57 44@2x" src="https://user-images.githubusercontent.com/15716057/183626822-3f2a548c-2f89-425b-83f0-392486636acb.png">

Reply:
Hiding content on smaller screen works.